core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -25,7 +25,7 @@ import java.util.Random
 import java.util.Properties
 
 import kafka.common.TopicAlreadyMarkedForDeletionException
-import org.apache.kafka.common.errors.{InvalidPartitionsException, InvalidReplicaAssignmentException, InvalidReplicationFactorException, InvalidTopicException, TopicExistsException, UnknownTopicOrPartitionException}
+import org.apache.kafka.common.errors._
```

Comment:
IDE, but I think it was triggered because it had to add `ReassignmentInProgressException` to the imports. Since I don't see it [mentioned](https://kafka.apache.org/coding-guide), what's the policy wrt wildcard imports?

Reply:
We disallow them in Java via checkstyle:

https://github.com/apache/kafka/blob/trunk/checkstyle/checkstyle.xml#L72

Same policy for Scala, but not enforced, which results in inconsistency.

Reply:
An exception to this is when using wildcard imports for the type class pattern. For example, `import scala.collection.JavaConverters._`.